modules/uhttp/handler.go

```diff
@@ -58,7 +59,8 @@ type handlerWrapper struct {
 // ServeHTTP calls Handler.ServeHTTP(ctx, w, r) and injects a new service context for use.
 func (h *handlerWrapper) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 	ctx := fx.NewContext(context.Background(), h.host)
-	stopwatch := h.host.Metrics().Timer("http." + r.Method + ".time").Start()
+	stopwatch := stats.HTTPMethodTimer.Timer(r.Method).Start()
```

Comment:
Consider whether you really want "getUser" to be the name of this metric. Would "latency" be a better name, with method as a dimension?